src/buttons/CopyToClipboard/index.test.js

```diff
@@ -0,0 +1,36 @@
+import { render, fireEvent } from '@testing-library/preact'
+import copyToClipboard from './../../utils/copy-to-clipboard'
```

Comment:
(nit) please always insert a single empty line between vendor imports and project imports (with vendor imports always at the top), like so: 

```js
import { render, fireEvent } from '@testing-library/preact';

import copyToClipboard from './../../utils/copy-to-clipboard';
import CopyToClipboard from './index';
```